backend/src/test/resources/testResultUpload/fhir-for-csv-with-flu-only.ndjson

```diff
@@ -1,540 +1 @@
[{"use":"official","type":{"coding":[{"system":"http://loinc.org","code":"81959-9","display":"Public health laboratory ask at order entry panel"}]}}],"status":"final","code":{"coding":[{"system":"http://loinc.org","code":"82810-3","display":"Pregnancy status"}],"text":"Pregnancy status"},"subject":{"reference":"Patient/1234"},"valueCodeableConcept":{"coding":[{"system":"http://snomed.info/sct","code":"60001007","display":"Not pregnant"}]}}},{"fullUrl":"Observation/7cae2c8e-cc60-35f6-8980-dd6bc9e1f539","resource":{"resourceType":"Observation","id":"7cae2c8e-cc60-35f6-8980-dd6bc9e1f539","identifier":[{"use":"official","type":{"coding":[{"system":"http://loinc.org","code":"81959-9","display":"Public health laboratory ask at order entry panel"}]}}],"status":"final","code":{"coding":[{"system":"http://loinc.org","code":"95418-0","display":"Employed in a healthcare setting"}],"text":"Employed in a healthcare setting"},"subject":{"reference":"Patient/1234"},"valueCodeableConcept":{"coding":[{"system":"http://terminology.hl7.org/ValueSet/v2-0136","code":"N","display":"No"}]}}},{"fullUrl":"Observation/4403633f-899d-3042-8a3e-863c27bd57ae","resource":{"resourceType":"Observation","id":"4403633f-899d-3042-8a3e-863c27bd57ae","identifier":[{"use":"official","type":{"coding":[{"system":"http://loinc.org","code":"81959-9","display":"Public health laboratory ask at order entry panel"}]}}],"status":"final","code":{"coding":[{"system":"http://loinc.org","code":"77974-4","display":"Hospitalized for condition"}],"text":"Hospitalized for condition"},"subject":{"reference":"Patient/1234"},"valueCodeableConcept":{"coding":[{"system":"http://terminology.hl7.org/ValueSet/v2-0136","code":"N","display":"No"}]}}},{"fullUrl":"Observation/b903826a-06bc-3c2d-a0e1-3b9aed860f0a","resource":{"resourceType":"Observation","id":"b903826a-06bc-3c2d-a0e1-3b9aed860f0a","identifier":[{"use":"official","type":{"coding":[{"system":"http://loinc.org","code":"81959-9","display":"Public health laboratory ask at order entry panel"}]}}],"status":"final","code":{"coding":[{"system":"http://loinc.org","code":"95420-6","display":"Admitted to ICU for condition"}],"text":"Admitted to ICU for condition"},"subject":{"reference":"Patient/1234"},"valueCodeableConcept":{"coding":[{"system":"http://terminology.hl7.org/ValueSet/v2-0136","code":"N","display":"No"}]}}},{"fullUrl":"Observation/585761a1-2d41-3688-b5c7-9ae8535b4a67","resource":{"resourceType":"Observation","id":"585761a1-2d41-3688-b5c7-9ae8535b4a67","identifier":[{"use":"official","type":{"coding":[{"system":"http://loinc.org","code":"81959-9","display":"Public health laboratory ask at order entry panel"}]}}],"status":"final","code":{"coding":[{"system":"http://loinc.org","code":"95421-4","display":"Resides in a congregate care setting"}],"text":"Resides in a congregate care setting"},"subject":{"reference":"Patient/1234"},"valueCodeableConcept":{"coding":[{"system":"http://terminology.hl7.org/ValueSet/v2-0136","code":"N","display":"No"}]}}}]}
```

Comment:
can you format this and all of the json files?

Reply:
Formatted this and another one -- I left `backend/src/test/resources/testResultUpload/test-results-upload-valid-as-fhir.ndjson` file alone since the a test fails otherwise with the formatting

Thank you!! Let me know if I missed something!